app/services/change_schedule.rb

```diff
@@ -85,7 +85,7 @@ def cohort
   def relevant_induction_record
     return if user.blank? || participant_profile.blank?
 
-    @relevant_induction_record ||= participant_profile.latest_induction_record_for(cpd_lead_provider:)
```

Comment:
chatted in person: I was wondering if instead of changing the interface to call this new method, we can instead amend the implementation of `latest_induction_record_for`, and keep the interface everywhere the same? Also maybe we can consolidate `relevant_induction_record` and `latest_induction_record_for` if that's possible (not sure it is though)

Reply:
@cwrw  I've added these in, also when retrieving the schedule too.  I left a couple of `FIXME` comments in there too around confirming what should really happen if `cpd_lead_provider` is nil in those methods, if you know whether that's legit or not as it came up in the specs.